### ISSUE TITLE
[T10] P2 - e)

### ISSUE THREAD

**Issue by malopez16** [IIC2343/syllabus#257]
Hola!
En la pregunta e), obtenga la expresión (...), debemos obtenerla sólo para la imagen de 7x7? o para una imágen cualquiera? Si es una imagen cualquiera, tendremos sus dimensiones? Cómo la expresamos?

La pregunta surge debido a que en la b) sí se indica que tiene que ser para la imagen de 5x5 (por lo que asumo que la respuesta que buscan tiene que ser específica a ese porte y no para cualqueiera).
Gracias!

**Comment by Geeermy** [comment]
¡Hola!

La idea es que sea una expresión totalmente parametrizada, por lo que no estaría correcto obtenerla solo para las dimensiones 7x7.

Ahora, puedes asumir que tienes una función _sizeof(X)_ que retorna el tamaño de _X_ para no tener que dejar un valor fijo con respecto a la dimensionalidad. Te recomiendo revisar desde la página 11 de los apuntes de "Almacenamiento de Datos" para poder entender bien esa parte. 😁

**Comment by malopez16** [comment]
Hola! creo que me confundí . Con "Tamaño de X", pudiendo usar sizeof(x), a partir de tu comentario, entiendo que te estás refiriendo a la cantidad de pixeles por lado de la imagen (que asumimos que es cuadrada), pero leyendo el apunte, sizeof retorna el tamaño del tipo de dato del arreglo en bytes, que no es lo mismo

**Comment by Geeermy** [comment]
Cierto, toda la razón, perdón. De todas formas (asumiendo que almacenas la imagen de la forma esperada), tu expresión debería tener en alguna parte _sizeof(X)_, donde _X_ es el elemento de tu imagen y _sizeof(X)_ te retorna su tamaño en **bytes**. Ahora, si reemplazaste directamente por los _bytes_ utilizados, también está ok (siempre que quede claro).

Volviendo a tu pregunta original, como no se indican las dimensiones de la imagen en **2.e.**, puedes hacer uso de uno o más parámetros que representen dicha dimensionalidad (siempre indicando qué significa cada uno de ellos) para tu expresión, tales como _p_, _q_ _r_, _s_... la letra o símbolo que prefieras. Como se pide una expresión general, no hay problema con eso.

¡Éxito!

**Comment by malopez16** [comment]
Gracias , ahora sí me quedó claro :)
